Source/DataTypes/EnumConverters.cs

```diff
@@ -220,21 +220,21 @@ public sealed class SvgStitchTypeConverter : EnumBaseConverter<SvgStitchType> {
 
     public sealed class SvgTurbulenceTypeConverter : EnumBaseConverter<SvgTurbulenceType> { }
 
-    public static class Enums
-    {
-        [CLSCompliant(false)]
-        public static bool TryParse<TEnum>(string value, out TEnum result) where TEnum : struct, IConvertible
-        {
-            try
-            {
-                result = (TEnum)Enum.Parse(typeof(TEnum), value, true);
-                return true;
-            }
-            catch
-            {
-                result = default(TEnum);
-                return false;
-            }
-        }
-    }
+    //public static class Enums
+    //{
+    //    [CLSCompliant(false)]
+    //    public static bool TryParse<TEnum>(string value, out TEnum result) where TEnum : struct, IConvertible
+    //    {
+    //        try
+    //        {
+    //            result = (TEnum)Enum.Parse(typeof(TEnum), value, true);
+    //            return true;
+    //        }
+    //        catch
+    //        {
+    //            result = default(TEnum);
+    //            return false;
+    //        }
+    //    }
+    //}
```

Comment:
Is this not needed bevause you just use `Enum`? In this case just remove it. 
@H1Gdev may have a look as he introduced this.

Reply:
Thanks, we wait for @H1Gdev. 
The generic support for the parsing `enum` string was not available in 3.5, so that might be needed but no more.

Reply:
@paulushub @mrbean-bremen 

I agree with remove it.
It would be beneficial to be able to remove `[CLSCompliant(false)]`.

Reply:
@H1Gdev Thanks for the feedback. I will now remove the commented code.